docs/src/ensemble_kalman_inversion.md

```diff
@@ -1,37 +1,56 @@
 # Ensemble Kalman Inversion
 
-One of the ensemble Kalman processes implemented in `EnsembleKalmanProcesses.jl` is the ensemble Kalman inversion ([Iglesias et al, 2013](http://dx.doi.org/10.1088/0266-5611/29/4/045001)). The ensemble Kalman inversion (EKI) is a derivative-free ensemble optimization method that seeks to find the optimal parameters $\theta \in \mathbb{R}^p$ in the inverse problem
+One of the ensemble Kalman processes implemented in `EnsembleKalmanProcesses.jl` is the ensemble Kalman inversion ([Iglesias et al, 2013](http://dx.doi.org/10.1088/0266-5611/29/4/045001)). The ensemble Kalman inversion (EKI) is a derivative-free ensemble optimization method that seeks to find the optimal parameters ``\theta \in \mathbb{R}^p`` in the inverse problem
 
-\\[ y = \mathcal{G}(\theta) + \eta, \\]
+```math
+y = \mathcal{G}(\theta) + \eta,
+```
+
+where ``\mathcal{G}`` denotes the forward map, ``y \in \mathbb{R}^d`` is the vector of observations
+and ``\eta  \in \mathbb{R}^d`` is additive Gaussian observational noise. Here, we take 
+``\eta \sim \mathcal{N}(0, \Gamma_y)`` from a multivariate ``d``-dimensional normal distribution
+with zero mean and covariance matrix ``\Gamma_y``.  This noise structure aims to represent the
+uncertainty, including correlated uncertainty, in the observations. Note that ``p`` is the
+size of the parameter vector ``\theta`` and ``d`` is taken to be the size of the observation
+vector ``y``.
```

Comment:
repetition of definition p and d? perhaps just write this out when y and theta are defined?

"uncertainty, including correlated uncertainty, in the observations" -> "correlations between observations"